### ISSUE TITLE
How did you calculate the normalization constants?

### ISSUE THREAD

**Issue by ultimatedigiman** [stepankonev/waymo-motion-prediction-challenge-2022-multipath-plus-plus#1]
Hi,

I found some normalization constants in the repo, such as
* normalizing the future coordinates:
https://github.com/stepankonev/waymo-motion-prediction-challenge-2022-multipath-plus-plus/blob/cf020ffa5ea451389fc43d4fc460cb22ba2cc124/code/train.py#L85
* normalizing inputs:
https://github.com/stepankonev/waymo-motion-prediction-challenge-2022-multipath-plus-plus/blob/cf020ffa5ea451389fc43d4fc460cb22ba2cc124/code/prerender/utils/utils.py#L37-L61

I have a few questions about normalization, please let me know if you have any suggestions:
1. Is there any script to get these constants?
2. Have you tried training without any normalization? What's the result?
3. How to modify the training code to non-normalization version? Just set these 2 lines below to `False`?
https://github.com/stepankonev/waymo-motion-prediction-challenge-2022-multipath-plus-plus/blob/cf020ffa5ea451389fc43d4fc460cb22ba2cc124/code/configs/final_RoP_Cov_Single.yaml#L20-L21

**Comment by stepankonev** [comment]
Hi!
Thank you for your question. First, it seems like `get_normalize_data()` is outdated. It used to provide normalization during data preparation, however now it is done in dataloader. I will remove it in the following commits with the normalization option in prerender config.
As for your questions:
1. No script is provided for computing these constants yet.
2. Normalization seemed to provide numerical stability improvements. I did not train the network for long without normalization.
3. Exactly

Hope it was helpful

**Comment by ultimatedigiman** [comment]
Thanks! Steven
